gnovm/pkg/gnomod/utils.go

```diff
@@ -0,0 +1,24 @@
+package gnomod
+
+import (
+	"errors"
+	"os"
+	"path/filepath"
+)
+
+func GuessGnoModDir(absPath string) (string, error) {
```

Comment:
```suggestion
func GuessDir(absPath string) (string, error) {
```

Since usage is `gnomod.GuessDir` when imported.

Reply:
What about `FindRoot` or `FindProjectRoot`? (We're not "guessing", as we're not trying known paths, and I think `GuessDir` is not very descriptive)

Reply:
I agree. Changed it to `FindRootDir()`
See: f1432bffd2c7e7d21b4b00bfe5f197181dcf816d